docs/requirements/hardware-proposal.md

```diff
@@ -0,0 +1,98 @@
+# Hardware proposal
+
+Current document covers requirements in proposal real hardware solutions,
+which meet requirements gathered earlier.
+
+## Microchip ATTPM20P
+
+* Link to documentation: [mcp_attpm20p]:<https://ww1.microchip.com/downloads/en/DeviceDoc/ATTPM20P-Trusted-Platform-Module-TPM-2.0-SPI-Interface-Summary-Data-Sheet-DS40002082A.pdf>
+* Compliant to the Trusted Computing Group (TCG) Trusted Platform Module (TPM)
+Version 2.0, r116 Trusted Platform Module Library
+* Single-Chip Turnkey Solution
+* Hardware Asymmetric Crypto Engine
+* Microchip ARM® M0+Microprocessor
+* Internal FLASH Storage for Keys
+* Serial Peripheral Interface (SPI) Protocol up to 36 MHz
+* Secure Hardware and Firmware Design and Device Layout
+* FIPS-140-2 Module Compliant Including the High-Quality Random Number
+Generator (RNG), HMAC, AES, SHA, ECC, and RSA Engines
+
+* Offered in Commercial (0°C to +70°C) Temperature Range
+  * Supply Voltage: 1.8V to 3.3V
+* Offered in Industrial (-40°C to +85°C) Temperature Range
+  * Supply Voltage: 3.3V
+* Cryptographic Support for:
+  * HMAC
+  * AES-128
+  * SHA-1
+  * SHA-256
+  * ECC BN_P256, ECCNIST_P256
+  * RSA 1024-2048 bit keys
+* 16 KB of User-Accessible Nonvolatile Memory
+* X.509 EK Certificates (Optional)
+
+## Infineon CY8C6xxxx Series
+
+* Link to documentation: [Infineon PSoC6 Family]<https://www.infineon.com/cms/en/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/psoc-6-32-bit-arm-cortex-m4-mcu/>
```

Comment:
```suggestion
* Link to documentation: [Infineon PSoC6 Family](https://www.infineon.com/cms/en/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/psoc-6-32-bit-arm-cortex-m4-mcu/)
```

Reply:
(corrected, will be visible in this commit)